modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SelectionControls.js

```diff
@@ -127,13 +128,32 @@ const SelectionControls = ({
 				);
 			});
 
+			const allSelectedElements = select.getAllSelectedElements();
+
+			const elements = {
+				allSelectedElements,
+				currentPageElements: select._getCurrentPageElements(),
+				currentPageSelectedElements: select.getCurrentPageSelectedElements(),
+			};
+
+			if (isFirstRenderRef.current && allSelectedElements.size()) {
```

Comment:
Do we need this first render check? This code is in `useEffect` with no arguments and in `componentReady`, so all of it should already be executed only on first render.

Reply:
Quite right, I removed that part and force pushed the change.

Thanks,